introduction.md

```diff
@@ -1,25 +1,28 @@
-##সংক্ষিপ্ত ইতিহাস
+## সংক্ষিপ্ত ইতিহাস
 
-১৯৭০ দশকের প্রথম দিকে Donald D. Chamberlin এবং Raymond F. Boyce আইবিএমের "System R" নামক রিলেশনাল ডাটাবেজ এ ব্যাবহারের জন্য একটি লাংগুয়েজ তৈরি শুরু করেন। তারা লাংগুয়েজটির প্রথম ভার্সনটির নাম দেন SEQUEL (Structured English Query Language) যা পরবর্তিতে ট্রেডমার্ক জনিত সমস্যার কারনে পরিবর্তিত হয়ে SQL (Structured Query Language) হয়।
+১৯৭০ দশকের প্রথম দিকে [Donald D. Chamberlin](https://en.wikipedia.org/wiki/Donald_D._Chamberlin) এবং [Raymond F. Boyce](https://en.wikipedia.org/wiki/Raymond_F._Boyce) আইবিএমের "System R" নামক রিলেশনাল ডাটাবেজ এ ব্যাবহারের জন্য একটি লাংগুয়েজ তৈরি শুরু করেন। তারা লাংগুয়েজটির প্রথম ভার্সনটির নাম দেন SEQUEL \(Structured English Query Language\) যা পরবর্তিতে ট্রেডমার্ক জনিত সমস্যার কারনে পরিবর্তিত হয়ে SQL \(Structured Query Language\) হয়।
 
- এসকিউএল হচ্ছে Edgar F. Codd এর রিলেশনাল মডেল এর উপর ভিত্তি করে বানানো প্রথম ল্যাংগুয়েজ। যদিও এটি তখন তার রিলেশনাল মডেলের সাথে পুরোপুরি সামঞ্জস্য পূর্ন ছিল না। পরবর্তীতে এটিই বিশ্বের সবচেয়ে বহুল ব্যাবহৃত ডাটাবেজ লাংগুয়েজে পরিনত হয়।
+এসকিউএল হচ্ছে [Edgar F. Codd](https://en.wikipedia.org/wiki/Edgar_F._Codd) এর রিলেশনাল মডেল এর উপর ভিত্তি করে বানানো প্রথম ল্যাংগুয়েজ। যদিও এটি তখন তার রিলেশনাল মডেলের সাথে পুরোপুরি সামঞ্জস্য পূর্ন ছিল না। পরবর্তীতে এটিই বিশ্বের সবচেয়ে বহুল ব্যাবহৃত ডাটাবেজ লাংগুয়েজে পরিনত হয়।
 
-##এসকিউএল কেন শিখবেন ?
+## এসকিউএল কেন শিখবেন ?
 
```

Comment:
can we remove point 1,2,3?
I think they are not educational rather career advice.